rmc/utils/missense_badness.py

```diff
@@ -443,7 +461,7 @@ def calculate_misbad(
         mb_ht = mb_ht.select(high_low_col, mb_col)
         mb_ht = misbad_ht.annotate(**mb_ht[misbad_ht.key])
         # Checkpointing so that `misbad` can be overwritten to the same path
-        mb_ht = mb_ht.checkpoint(f"{TEMP_PATH_WITH_FAST_DEL}/misbad.ht")
+        mb_ht = mb_ht.checkpoint(f"{TEMP_PATH_WITH_FAST_DEL}/misbad.ht", overwrite=True)
         # Overwrite is set to True here to ensure newly-added columns are written out
         mb_ht.write(misbad.path, overwrite=True)
     else:
```

Comment:
I can't comment on the line below, but it should also have the option for overwrite set to overwrite_output, right?

Reply:
Yes, thank you for catching these